index.html

```diff
@@ -39,14 +46,548 @@ <h1>I'm Emeka Glad to see you!</h1>
             <h5>LET'S CONNECT</h5>
 
             <ul>
-                <li><a href="#"><img src="./img/angular.jpg" alt="angular_image"></a></li>
-                <li><a href="#"><img src="./img/github.jpg" alt="github_image"></a></li>
-                <li><a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a></li>
-                <li><a href="#"><img src="./img/mee.jpg" alt="mee_image"></a></li>
-                <li><a href="#"><img src="./img//twitter.jpg" alt=""></a></li>
+
+                <li>
+
+                    <a href="#"><img src="./img/angular.jpg" alt="angular_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/github.jpg" alt="github_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/mee.jpg" alt="mee_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img//twitter.jpg" alt="twitter_image"></a>
+
```

Comment:
- [ ] Please remove these whitespaces between your codes throughout this file because they make your code difficult to read and not clean 😃

Reply:
Hi Betrand, my last reviewer asked for white spacing with the idea, it improves overall coding experience. With this conflicting views from yours. How do i know what is right?...or is this an arbitary judgement?